graphql/src/store/query.rs

```diff
@@ -100,11 +100,18 @@ fn list_values(value: Value) -> Vec<Value> {
 }
 
 /// Parses GraphQL arguments into an attribute name to order by, if present.
-fn build_order_by(arguments: &HashMap<&q::Name, q::Value>) -> Option<String> {
+fn build_order_by(
```

Comment:
This function needs to return a result and so should `build_query`. Because if the query is somehow not quite right, we want to send an error back to the client.